beamer/agent/data/agent.conf.example

```diff
@@ -15,6 +16,7 @@ prometheus-port = 9101
 [base-chain]
 rpc-url = "GOERLI_TESTNET_RPC_URL"
 poll-period = 60.0
```

Comment:
Config seems wrong to me, base-chain shouldnt have any options like poll-period or min-source-balance instead we should use normal chains.